src/Pos/Launcher/Runner.hs

```diff
@@ -220,6 +223,7 @@ runServer mkTransport mkReceiveDelay mkL (OutSpecs wouts) withNode afterNode oq
             mkListeners mkL' ourVerInfo theirVerInfo
     stdGen <- liftIO newStdGen
     logInfo $ sformat ("Our verInfo: "%build) ourVerInfo
+    notifyReady
```

Comment:
On second though maybe the proper spot for it is in `src/Pos/Launcher/Scenario.hs`, just before the workers start to run. Anyway, somebody else could make that change if needed. Enjoy your holiday!